improver/utilities/mathematical_operations.py

```diff
@@ -270,8 +296,11 @@ def perform_integration(
                        self.end_point, self.direction_of_integration))
             raise ValueError(msg)
 
-        # Merge resulting cubes back together
-        integrated_cube = integrated_cubelist.merge_cube()
+        template = (upper_bounds_cube.copy()
+                    if self.direction_of_integration == "positive" else
+                    lower_bounds_cube.copy())
+        integrated_cube = self._create_output_cube(
+            template, data, coord_points, coord_bounds)
         return integrated_cube
 
     def process(self, cube):
```

Comment:
This doc-string needs updating as the list of functions utilised is no longer correct.

Reply:
Yes.  Actually I need to make another change to the plugin.  I've added a "TODO" to this docstring, but we should discuss (I think this should actually be done in this PR but it's not ideal...)